imagetest/cmd/manager/main.go

```diff
@@ -199,6 +215,12 @@ func main() {
 		},
 	}
 
+	// These tests are excluded by default so they do not block releases.
+	skippedNonblockingTests := make(map[string]bool)
+	for _, skippedNonblockingTestName := range []string{fioperf.Name} {
```

Comment:
It may be a good idea to make the array of blocking test names a module-level variable. That way it'll be easier to find and update the list of tests whenever a new one is added.

Reply:
the test suites which are excluded are now in the concourse files